project/eScavenge/CLI.py

```diff
@@ -128,9 +128,11 @@ def end(self, _):
 
 @need_admin
 def create(self, _):
+    if self.game.started or not self.game.ended:
+        return Errors.GAME_IN_PROGRESS
     try:
         self.game = GameFactory(make_game).create_game()
-    except IndexError:
+    except ValueError:
```

Comment:
ValueError is also meaningless I think. I think the try/except can go away completely, there aren't any exceptions that can be thrown that I know of...

Reply:
Shouldn't there be some kind of indicator of whether or not we are getting a game created or not though? I was perusing the exception list and couldn't find anything relevant.  If that's the case then we should change the start and end commands as well as they are currently using the indexerror exception as well

Reply:
GameFactory cannot return anything except a new game. So if it completes successfully then we will get a new game. There are currently no cases in our code that will prevent it from running successfully. So the only possible exceptions we could get are completely random system errors (out of memory, system unstable, and the like) that would kill every other call we try and make most likely. So there's nothing for us to catch.
You are probably correct on the start and end, since we don't pass anything to them there's nothing (that we introduce) that could cause an exception.

Reply:
For try/excepts in that location the intent was to catch int("abc") [ValueError] when the user puts in a string and we expect an int. or args[i] when i doesn't exist [IndexError] when the user misses a parameter.